### PR TITLE
Upgrade dependency for security patch

### PR DESCRIPTION
**Pull request by toinoudivad**
### Purpose/goal of Pull Request (Issue #)
For this: https://github.com/WOVNio/wovnrb/network/dependencies#34578148

### CODE REVIEW

File: wovnrb.gemspec [hunk 1]
```diff
@@ -26,7 +26,7 @@ Gem::Specification.new do |spec|
   #spec.extensions    = spec.files.grep(%r{/extconf\.rb$})
 
   spec.add_dependency "nokogumbo", ">= 1.3.0"
-  spec.add_dependency "nokogiri", "<= 1.8"
+  spec.add_dependency "nokogiri", "<= 1.8.1"
```
Comment (yeuem1vannam) [inline wovnrb.gemspec]: Is it still under the range of security issue?
image

Comment (toinoudivad) [reply wovnrb.gemspec]: As I understand "update suggested" means it will fix the vulnerability found, what do you understand?

Comment (yeuem1vannam) [reply wovnrb.gemspec]: Yes I understand so, too.
So it means we should update the dependency to `~> 1.8.1`

Comment (toinoudivad) [reply wovnrb.gemspec]: ahhh got it, I will do it thanks lol

**Comment by toinoudivad** [comment]
Need to hold this until we can upgrade Nokogiri version in equalizer too

**Comment by yeuem1vannam** [comment]
Should we merge this PR with version pump 0.2.31, and lock the version of `wovnrb` in equalizer at 0.2.30

**Pull request merged**